Makefile

```diff
@@ -409,7 +412,10 @@ typos_check:
 
 check: initramfs format_check workspace_project_coverage_check lint_check clippy_check c_code_check typo_check
 
-.PHONY: clean
+docker:
+	docker run -it --privileged --network=host --device=/dev/kvm -v $(shell pwd):/root/asterinas ldosproject/asterinas:0.15.2-20250613
```

Comment:
1. This will not play nice with `make -C`. That's probably OK, but it's not ideal.
2. This hardcodes the version. Use the version number of [DOCKER_IMAGE_VERSION](https://github.com/ldos-project/asterinas/blob/9553180554da70935c9413e07d3a6f43dd383186/DOCKER_IMAGE_VERSION).
3. I think this should set the container name. The reason is that this will leave the container running. I think that is the correct way to do it, but it can result in a bunch of copies of the container. If you set the name, starting it again will produce an error. Maybe having a `make attach_docker` would be a good idea to do the `exec bash -i` thing.